packages/ws-server/tests/acceptance/estimateGas.spec.ts

```diff
@@ -51,6 +53,26 @@ describe('@release @web-socket eth_estimateGas', async function () {
 
   beforeEach(async () => {
     ethersWsProvider = new ethers.WebSocketProvider(WsTestConstant.WS_RELAY_URL);
+    const wallet = new ethers.Wallet(accounts[0].wallet.privateKey, ethersWsProvider);
+    const basicContractFactory = await new ethers.ContractFactory(
+      basicContractJson.abi,
+      basicContractJson.bytecode,
+      wallet,
```

Comment:
Q: Can we simply use `accounts[0].wallet` here?

Reply:
Yes. Updated.

Reply:
I am using the `wallet` created using the `ethWsProvider`.  See comment above.

Reply:
Yep totally make sense! Thanks!